### PR TITLE
Add style guidelines on multi-lining

### PR DESCRIPTION
**Pull request by Mothblocks**
We still don't require it, but when you do, this is the style we have been following.

### CODE REVIEW

File: .github/guides/STYLE.md [hunk 1]
```diff
@@ -400,6 +400,28 @@ turn_on(power_usage = 30) // Fine!
...
+## Multi-lining
+
+Whether it's a very long proc call, a long list people will be adding to, or something else entirely, there may be times where splitting code across multiple lines is the most readable. When you have to is up to maintainer discretion, but if you do, follow this consistent style.
+
+```dm
+proc_call_on_one_line(
+	arg1, // Only indent once! Remember to not align tabs.
+	arg2,
+	arg3, // End with a trailing comma
+) // The parenthese should be on the same indentation level as the proc call
```
Comment (Mothblocks) [inline .github/guides/STYLE.md:412]:
```suggestion
) // The parenthesis should be on the same indentation level as the proc call
```

**Review by MrMelbert — COMMENTED** [review]
We should probably have a honorable mention for macros like `AddComponent`, as we don't have a consistent method of multi-lining it 

image

image

image

image

**Comment by Mothblocks** [comment]
You're right

**Comment by Mothblocks** [comment]
That also reminds me that I'm generally okay with `list(` being on its own line in an argument where it makes sense, like in AddComponent.

**Comment by Mothblocks** [comment]
Doen

**Pull request merged**